packaging/installer/dependencies/debian.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+# Package tree used for installing netdata on distribution:
+# << Debian >>
```

Comment:
Also Raspbian, probably also Devuan, and possibly other non-Ubuntu Debian derivatives (such as LMDE).